docs/src/__examples__/ButtonPrimitive/DEFAULT.tsx

```diff
@@ -14,9 +14,142 @@ export default {
       Click me
     </ButtonPrimitive>
   ),
-  info: {
-    title: "Default button primitive",
-    description:
-      "The default button primitive appears as a &lt;button&gt; with other traits inherited or as the browser default.",
-  },
+  exampleKnobs: [
+    {
+      component: "ButtonPrimitive",
+      knobs: [
+        {
+          name: "iconLeft",
+          type: "icon",
+          defaultValue: "",
+        },
+        {
+          name: "iconRight",
+          type: "icon",
+          defaultValue: "",
+        },
+        {
+          name: "icons",
+          type: "icon",
+          defaultValue: "",
+        },
+        {
+          name: "submit",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "fullWidth",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "title",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "height",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "boxShadow",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "boxShadowHover",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "boxShadowActive",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "boxShadowFocus",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "contentWidth",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "circled",
+          type: "boolean",
+          defaultValue: false,
+        },
```

Comment:
For some reason this knob works in reverse; turning it on makes it a square, and turning it off makes the corners rounded (not circled, not sure what the intention is).

Reply:
that's because height is not set. Good catch 👍🏻